### ISSUE TITLE
return numpy array instead of torch tensor (optionally?) 

### ISSUE THREAD

**Issue by apoorvalal** [apoorvalal/covariate_balancing_propensity_scores#3]
add `numpy` argument to `weights` method that calls `.detach().numpy()`
